src/main/java/net/darksky/project/numberone/EnterInput.java

```diff
@@ -0,0 +1,28 @@
+package net.darksky.project.numberone;
+
+import java.util.Scanner;
+
+public class EnterInput {
+	private Scanner scanner = new Scanner(System.in);
+	private double latitude;
+	private double longitude;
+
+	public double getLatitude() {
+		this.latitude = scanner.nextDouble();
+		if (latitude < 90 && latitude > 90) {
```

Comment:
Is `latitude` always 90?

Reply:
This is why you need unit tests.

Reply:
Latitude must be between -90 and 90, but Longitude -180 and 180